core/tests/webdriverio_utils/PreferencesPage.js

```diff
@@ -49,6 +49,8 @@ var PreferencesPage = function() {
   var userInterestsInput = $('.e2e-test-subject-interests-input');
 
   var saveNewChanges = async function(fieldName) {
+    await action.click('Navbar Button', navBar);
```

Comment:
What is the "navbar button" and why do we need to click it?

If this is the "neutral element" thing then you should explain so that devs know what is going on and what happens if it is removed.

Reply:
Done.